graveler/staging_test.go

```diff
@@ -118,6 +116,158 @@ func TestDrop(t *testing.T) {
 	}
 }
 
+func TestDropByPrefix(t *testing.T) {
+	s := newTestStagingManager(t)
+	numOfValues := 2400
+	for i := 0; i < numOfValues; i++ {
+		err := s.Set(context.Background(), "t1", []byte(fmt.Sprintf("key%04d", i)), newTestValue(fmt.Sprintf("identity%d", i), fmt.Sprintf("value%d", i)))
+		if err != nil {
+			t.Fatalf("got unexpected error: %v", err)
+		}
+		err = s.Set(context.Background(), "t2", []byte(fmt.Sprintf("key%04d", i)), newTestValue(fmt.Sprintf("identity%d", i), fmt.Sprintf("value%d", i)))
+		if err != nil {
+			t.Fatalf("got unexpected error: %v", err)
+		}
+	}
+	err := s.DropByPrefix(context.Background(), "t1", []byte("key1"))
+	if err != nil {
+		t.Fatalf("got unexpected error: %v", err)
+	}
+	v, err := s.Get(context.Background(), "t1", []byte("key1000"))
+	if !errors.Is(err, db.ErrNotFound) {
+		// key1000 starts with the deleted prefix - should have been deleted
+		t.Fatalf("after dropping staging area, expected ErrNotFound in Get. got err=%v, got value=%v", err, v)
+	}
+	v, err = s.Get(context.Background(), "t1", []byte("key0000"))
+	if err != nil {
+		// key0000 does not start with the deleted prefix - should be returned
+		t.Fatalf("got unexpected error: %v", err)
+	}
+	it, _ := s.List(context.Background(), "t1")
+	count := 0
+	for it.Next() {
+		count++
+	}
+	it.Close()
+	if count != numOfValues-1000 {
+		t.Errorf("got unexpected number of results after drop. expected=%d, got=%d", numOfValues-1000, count)
+	}
+	it, _ = s.List(context.Background(), "t2")
+	count = 0
+	for it.Next() {
+		count++
+	}
+	it.Close()
+	if count != numOfValues {
+		t.Errorf("got unexpected number of results. expected=%d, got=%d", numOfValues, count)
+	}
+}
+
+func TestDropPrefixBytes(t *testing.T) {
```

Comment:
usually, when you use idea to create a test function it will also add 'name' as the test parameter which helps describe the use case we are testing. in this case, it looks like it will be t<number> which is less descriptive.

Reply:
use t.Run with name